steps/s3-cache-push/0.1.0/step.yml

```diff
@@ -0,0 +1,104 @@
+title: S3 Cache Push
+summary: |
+  Store your cache in a s3 bucket with custom keys.
+website: https://github.com/alephao/bitrise-step-s3-cache-push
+source_code_url: https://github.com/alephao/bitrise-step-s3-cache-push
+support_url: https://github.com/alephao/bitrise-step-s3-cache-push/issues
+published_at: 2021-06-01T13:13:10.95948-03:00
+source:
+  git: https://github.com/alephao/bitrise-step-s3-cache-push.git
+  commit: 129259c7e76fafd22d19a40d4baf6077613fee9a
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/alephao/bitrise-step-s3-cache-push
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- aws_access_key_id: $AWS_ACCESS_KEY_ID
```

Comment:
Please don't use default env var values when it is not expected to be available. (exported by other steps as outputs or prefilled by bitrise webhook handling). This applies to the rest of the inputs too.